src/index.js

```diff
@@ -236,17 +236,9 @@ const sortLogic = function (key1, key2) {
 const sortAttr = (tree, {rules: {sortAttr}}) => {
   tree.map(node => {
     if (sortAttr && node.attrs) {
-      var keys = Object.keys(node.attrs);
-      if (keys.length > 1) {
-        keys = keys.sort(sortLogic);
-        const newAttrsObject = {};
-
-        keys.forEach(k => {
-          newAttrsObject[k] = node.attrs[k];
-        });
-
-        node.attrs = newAttrsObject;
-      }
+      node.attrs = Object.keys(node.attrs)
```

Comment:
I think we can keep the `keys.length` check cause if there are only one attribute, we dont need to sort and use the reduce for that .

what do you think

Reply:
I agree this is a good performance

Reply:
yea